src/WebApi/Infrastructure/Migrations/20220202082648_InitalCreate.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Infrastructure.Migrations
 {
-    public partial class InitialCreate : Migration
+    public partial class InitalCreate : Migration
```

Comment:
Should this be rather `InitialCreate` ? 🤔

Reply:
Fixed